### ISSUE TITLE
[Feature] Group focus areas in journey details

### ISSUE THREAD

**Issue by hildeor** [AtB-AS/mittatb-app#630]
## Origin
_Links to received feedback, user research or other findings._
[Usability test with Blindeforbundet, 29.10.2020](https://github.com/AtB-AS/innsikt/blob/master/brukertester/2020-10-29_Blindeforbundet.md#13):

> I Reisedetaljer har avgangstid og sted ulike fokusområder. Gjør at man må klikke unødvendig mange ganger for å få konusmert informasjon som egentlig hører sammen.

## Motivation
_A short description of what user needs or business goals this feature will solve._

## Hypotheses and assumptions
_A list of hypotheses and assumptions we have made about the user or the proposed solution._

## Proposed solution
_A coarse description of a proposed solution, that may include wireframes and graphic design._

Time stamps with corresponding info should be grouped together with a joint focus area. The step for the specific bus leg already has this. 
![fokusområder_reisedetaljer](https://user-images.githubusercontent.com/8980523/97855383-ee478b80-1cfa-11eb-995d-043a2f437bab.jpeg)

**Comment by hildeor** [comment]
Fixed via #622
